Source/Core/Core/PowerPC/PPCAnalyst.cpp

```diff
@@ -105,7 +105,7 @@ bool AnalyzeFunction(u32 startAddr, Common::Symbol& func, u32 max_size)
     }
     const PowerPC::TryReadInstResult read_result = PowerPC::TryReadInstruction(addr);
     const UGeckoInstruction instr = read_result.hex;
-    if (read_result.valid && PPCTables::IsValidInstruction(instr))
```

Comment:
I think that `IsValidInstruction` is a good function name worth keeping around, even if only to avoid extra diffs. The body would just be `GetOpID(instr) != OpID::Invalid` of course, but I think the clarity of the name is worth it.